docs/QuestionsIssuesPrs.md

```diff
@@ -0,0 +1,109 @@
+# Workflow for submitting Questions, Issues and PRs
+
+* [Asking a Question](#asking-a-question)
+* [Posting an Issue](#posting-an-issue)
+* [Submitting a PR](#submitting-a-pr)
+
+## Asking a Question
+
+Post your question in the issue tracker.<br>
+See [Posting an Issue](#posting-an-issue) for guidelines to follow when posting an issue.
+
+If you don't get an answer within 1-2 days, feel free to post another comment in the issue thread as a reminder.
+
+
+## Posting an Issue
+
+1. Do a search in the issue tracker to ensure the same issue has not been posted before.
+2. Follow the [naming convention](FormatsAndConventions.md#issue) for the issue title. 
+
+
+## Submitting a PR
+
+> This is an adaptation of the [_GitHub Flow_](https://guides.github.com/introduction/flow/). 
+
+Note that each project might specify additional things to do. i.e. follow these steps in conjunction with 
+additional details provided by the specific project.
+
+1. Read any instructions the project has for new contributors. Some projects require you to read some documents
+   and/or do some orientation activities before you start contributing. 
+   For example, [TEAMMATES](../../../teammates/teammates) has a 'Contributor Orientation Guide'.
```

Comment:
1. Using relative may work, but it should be better with absolute.
2. Also, may want to link to TEAMMATES' contributor orientation guide.